public/userInfo.php

```diff
@@ -25,6 +25,10 @@
     abort(404);
 }
 
+if ((int) $userMassData['Permissions'] < Permissions::Registered && $permissions < Permissions::Admin) {
+    abort(404);
+}
```

Comment:
An Unregistered user should be allowed to access their own page.

Reply:
Oh my ... thanks for spotting that!